### PR TITLE
strace 4.24

### PR DESCRIPTION
**Pull request by maxim-belkin**

### CODE REVIEW

File: Formula/strace.rb [hunk 1]
```diff
@@ -17,11 +17,14 @@ class Strace < Formula
   depends_on "linux-headers"
 
   def install
+    args = %W[
+      --prefix=#{prefix}
+      --disable-dependency-tracking
+      --disable-silent-rules
+      --enable-mpers=check
+    ]
```
Comment (sjackman) [inline Formula/strace.rb]: No need to use `args`. Please leave it as it was, inline.

File: Formula/strace.rb [hunk 2]
```diff
@@ -17,11 +17,14 @@ class Strace < Formula
   depends_on "linux-headers"
 
   def install
+    args = %W[
+      --prefix=#{prefix}
+      --disable-dependency-tracking
+      --disable-silent-rules
+      --enable-mpers=check
```
Comment (sjackman) [inline Formula/strace.rb]: Why have you added this option `--enable-mpers=check`, and what does it do? Please add an explanation to the commit log message and pull request description.

**Review by sjackman — CHANGES_REQUESTED** [review]

File: Formula/strace.rb (hunk 2 above)
Comment (maxim-belkin) [reply Formula/strace.rb]: the default value is `yes` and formula fails to build. With `check` the build succeeds and I get the following:

```
checking for m32 personality compile support... yes
checking for m32 personality runtime support... no
```

File: Formula/strace.rb (hunk 1 above)
Comment (maxim-belkin) [reply Formula/strace.rb]: I sort of disagree but also don't care that much at the same time, so ok! :)

File: Formula/strace.rb (hunk 2 above)
Comment (sjackman) [reply Formula/strace.rb]: Please add a comment like `Disable m32 personality support`
And I'd suggest `--enable-mpers=no` for reproducible builds.
See https://github.com/strace/strace/blob/150655ca3e0f68f92dcb1e434301ac485aad5e15/configure.ac#L927

Comment (sjackman) [reply Formula/strace.rb]: Please post the build error message to this PR.

Comment (sjackman) [reply Formula/strace.rb]: You could add to the formula `# Fix ERROR-MESSAGE-GOES-HERE`

**Comment by maxim-belkin** [comment]
gist-log of a build failure with enabled mpers: https://gist.github.com/7cc60fba21247db8c7cdda84f98b0b71

File: Formula/strace.rb (hunk 2 above)
Comment (maxim-belkin) [reply Formula/strace.rb]: posted log here: https://github.com/Linuxbrew/homebrew-extra/pull/38#issuecomment-413984644

**Review by sjackman — APPROVED** [review]

**Comment by sjackman** [comment]
Thanks, Maxim!

**Pull request closed without merging**